CoreQualityControlView/src/au/gov/asd/tac/constellation/views/qualitycontrol/daemon/QualityControlAutoVetter.java

```diff
@@ -192,7 +193,7 @@ public void run() {
             thread.start();
 
             try {
-                countDownLatch.await();
+                countDownLatch.await(1, TimeUnit.SECONDS);
```

Comment:
@aldebaran30701 apologies for the delay getting back to you on this PR.

I'd like to see if we can dig deeper into what the root cause might be following on from your findings from https://github.com/constellation-app/constellation/issues/170#issuecomment-570992326

I think I've introduced this bug so happy to look at assisting with a fix. I'm just having trouble getting reproducing this so I might have to introduce a sleep in one of the classes to force the freeze and maybe use that to see what other classes are getting blocked.

If you get to this before I do then that is also fine and welcomed 😄

Reply:
#214 is now the most relevant Pull Request for this issue. Close this if necessary.

Reply:
Thanks @aldebaran30701